networking/ipsec/ipsec_basic/ipsec_basic_netns/runtest.sh

```diff
@@ -190,7 +190,7 @@ rlJournalStart
 
 	rlPhaseStartTest "$SUB_PARAM"
 		data_tests
-		perf_tests
+		#perf_tests
```

Comment:
Could you add a comment stating the reason why this is commented out? If I was reading the code and stumbled upon this line I would assume it was a leftover and my first reaction would be to delete it.

The alternative is actually deleting the line instead of commenting it out. Which is ok with me, but maybe leaving it along with an explanatory comment is clearer? I'm thinking that if someone ever noticed that the line is missing (eg. if they compare the code to the "upstream" version) maybe they will think the line was added afterwards and thus they would be improving the test by re-adding it...

What do you think?

Reply:
LGTM, maybe just add the comment as @emanchado and we can merge :-)

Reply:
I agree with you, and I updated the new patch , please help review. Thanks